llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -201,9 +204,18 @@ def save_model_patch(*args: Any, **kwargs: Any):
     if mlflow_logger.model_registry_uri is not None:
         mlflow.set_registry_uri(mlflow_logger.model_registry_uri)
 
+    if is_peft:
+        transformers_in_memory_model = pipeline(
+            model=AutoModel.from_pretrained(
+                transformers_model,
+            )
+        )
+    else:
+        transformers_in_memory_model = None
+
     register_model_path = f'{mlflow_logger.model_registry_prefix}.{registered_model_name}' if mlflow_logger.model_registry_prefix and registered_model_name else registered_model_name
     mlflow_logger.log_model(
-        transformers_model=transformers_model,
+        transformers_model=transformers_in_memory_model or transformers_model,
```

Comment:
pretty sure you should just be passing the model and tokenizer in here directly as
{'model': model, 'tokenizer': tokenizer}, since mlflow doesn't support local paths for peft models.

Reply:
Ah I'll try this.